packages/flutter_tools/lib/src/localizations/gen_l10n.dart

```diff
@@ -1141,6 +1148,22 @@ class LocalizationsGenerator {
       );
     });
 
+    // Warn users when placeholder types are overrided to 'num'.
```

Comment:
This comment probably isn't necessary; at the moment it doesn't describe the entire block.